python-client/giskard/client/project.py

```diff
@@ -1,21 +1,13 @@
-import json
-from typing import Callable, Dict, Iterable, List, Optional, Union
+from typing import Callable, Iterable, Union
 
-import cloudpickle
-import numpy as np
 import pandas as pd
-import requests
-from pandas.api.types import is_numeric_dtype
-from pandas.api.types import is_string_dtype
 from requests_toolbelt.sessions import BaseUrlSession
 
 from giskard.client.analytics_collector import GiskardAnalyticsCollector, anonymize
-from giskard.client.io_utils import compress, pickle_dumps, save_df
-from giskard.client.model import SupportedColumnType, SupportedModelTypes
-from giskard.client.python_utils import get_python_requirements, get_python_version, warning
+from giskard.client.io_utils import compress, pickle_dumps
 
 
-class GiskardProject:
+class Project:
```

Comment:
we wanted to rename GiskardDataset and GiskardModel to simply Dataset and Model since some time ago, I took a chance to rename projects too, no more rational than that